conf/modules.config

```diff
@@ -846,7 +862,7 @@ process {
     withName: '.*ANNOTATE_SNVS:BCFTOOLS_VIEW' {
         ext.when = { params.annotate_snv_switch }
         ext.prefix = { "${meta.id}_view" }
-        ext.args = { '--output-type z --exclude "INFO/GNOMADAF > 0.70 | INFO/GNOMADAF_popmax > 0.70  | INFO/SWEGENAF > 0.70" ' }
+        ext.args = { '--output-type z' }
```

Comment:
I get swegen, but why do you want to remove gnomad?

Reply:
I had to remove that because we are not using gnomad to annotate the test dataset. So bcftools view was complaining about missing headers. :)

Reply:
Ah OK, It would be better to add this to a test config instead and have the original config here.

Reply:
Good point 👍🏻